### PR TITLE
Intellij: Explicitly set EXIST_HOME during startup of server and client;

### PR DESCRIPTION
**Pull request by dizzzz**
Now the expath repo is found at startup, and XAR files can executed

### CODE REVIEW

**Comment by dizzzz** [comment]
anyone?

**Comment by joewiz** [comment]
This is intellij-specific, right?  So ordinary users who build from source via ant wouldn't be able to test this, right?

**Comment by dizzzz** [comment]
correct...

File: .idea/runConfigurations/Client.xml [hunk 1]
```diff
@@ -3,15 +3,17 @@
     <extension name="coverage" enabled="false" merge="false" sample_coverage="true" runner="idea" />
     <option name="MAIN_CLASS_NAME" value="org.exist.client.InteractiveClient" />
     <option name="VM_PARAMETERS" value="" />
-    <option name="PROGRAM_PARAMETERS" value="" />
+    <option name="PROGRAM_PARAMETERS" value="-l" />
```
Comment (adamretter) [inline .idea/runConfigurations/Client.xml]: Did you really mean to set the client to embedded mode by default as well in this commit? If you did, it suits me :-)

**Pull request merged**